src/tokens/TokenImport.tsx

```diff
@@ -109,24 +109,21 @@ export default function TokenImportScreen(_: Props) {
       publicClient: client,
     })
 
-    const [symbol, decimals, name, balance] = await Promise.all([
```

Comment:
i think this balance check was done here not to store the balance, but to check that the balance can be correctly fetched (to avoid storing the token and getting errors later on). that way we can fail more quickly and show an error / not save this token. is there harm in keeping this?

Reply:
Hmm, I guess it's fine to fetch the balance here, I removed it because I feel it was really needed, but I can rollback this.